networks/suzuka/suzuka-full-node/src/partial.rs

```diff
@@ -169,43 +162,12 @@ where
 				}
 			};
 
-			let block: Block = serde_json::from_slice(&block_bytes)?;
+			let span = info_span!("execute_block", id = block_id);
```

Comment:
This needs a companion in the `m1-da-light-node`.

Reply:
Is this async block of interest: https://github.com/movementlabsxyz/movement/blob/497ec0f388596efdf837b48a9aa71accebbc1e78/protocol-units/da/m1/light-node/src/v1/sequencer.rs#L60?

Reply:
I would `span` the `wait_for_block` and the `submit_block`. Then, I would also at least log here: 

https://github.com/movementlabsxyz/movement/blob/497ec0f388596efdf837b48a9aa71accebbc1e78/protocol-units/da/m1/light-node/src/v1/passthrough.rs#L163

Reply:
If we want to have a span for waiting at the beginning in `tick_block_proposer`, what could be the identifying information? Can we anticipate the next height, for example?

Reply:
I have added an instrumentation span around the `LightNodeV1::get_blobs_at_height` method, which also logs a debug level event with each blob's ID. Hopefully this gives enough visibility from the passthrough flow.